src/execution.rs

```diff
@@ -24,6 +24,16 @@ pub enum Status {
     Skipped(String),
 }
 
+impl Status {
+    pub fn is_success_or_skip(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_ok(&self) -> bool {
```

An API doc would also be great.